spec/app/domain/util/concurrency_limited_cache_spec.rb

```diff
@@ -0,0 +1,141 @@
+require 'spec_helper'
+
+TARGET_EXCEPTION = "dummy exception"
+
+class ConcurrencyCount
+
+  attr_accessor :threads_blocked
+  attr_accessor :threads_failed
```

Comment:
Not a big deal but you can change these two lines to `attr_writer: :threads_blocked, :threads_failed`.